blobstore/src/main/java/org/jclouds/blobstore/config/LocalBlobStore.java

```diff
@@ -739,14 +738,8 @@ public Blob getBlob(String containerName, String key, GetOptions options) {
       Payload payload;
       try {
          InputStream is = blob.getPayload().openStream();
-         if (is instanceof FileInputStream) {
-            // except for FileInputStream since large MPU can open too many fds
-            is.close();
```

Comment:
A subsequent PR will return a `FileInputStream` instead of a `ByteSource` to fix a race condition in the filesystem blobstore so this code cannot assume that the `Payload` is a repeatable `ByteSource` that can be re-opened.